mono/metadata/class-internals.h

```diff
@@ -742,6 +742,9 @@ typedef struct {
 
 extern MonoPerfCounters *mono_perfcounters;
 
+#if ENABLE_NETCORE
+gint32 thread_contentions; /* for Monitor.LockContentionCount */
+#endif
```

Comment:
1. this doesn't need to be in a header, nothing outside monitor.c uses it.
2. if it _were_ ok to be in a header, it would need: (a) `extern` here (b) non-`extern` in monitor.c, (c) named with a `mono_` prefix.

So just make it a `static gint32` in `monitor.c`